actions/imanage/functions/imanage/1.0/index.js

```diff
@@ -0,0 +1,319 @@
+import lodash from "lodash";
+
+const BASEURL = "https://cloudimanage.com/work/api/v2/customers"
+
+const MODEL_LOOKUP = {
+    allDocuments: {
+        url: `${BASEURL}/{customerId}/documents`,
+        method: "GET",
+        processResponse: (respose) => {
```

Comment:
define the functios for processResponse() and returnFormat() and reuse them. Then the object will be something like;

url: `${BASEURL}/{customerId}/workspaces`,
        method: "GET",
        processResponse,
        returnFormat,
   }
   
   You could also using object spreading which would be even cleaner